File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n)
    {
       
        int ans{};
        int onestep=1;
        int twostep=1;
        if(n<=1)
        {
            return n;
        }
        for(int i=2;i<=n;i++)
        {
            ans=twostep+onestep;
            twostep=onestep;
            onestep=ans;
        }
        return ans;
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        arr=[0]*(n+1)
        arr[0]=1
        arr[1]=1
        for i in range(2,n+1):
            arr[i]=arr[i-1]+arr[i-2]
        return arr[n]